src/psqlgraph/poly.py

```diff
@@ -0,0 +1,52 @@
+"""A module for poly edges/nodes which generate an entity based on a label."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from psqlgraph import edge, node
+
+
+def PolyEdge(
+    *,
+    label: str,
+    src_id: str | None = None,
+    dst_id: str | None = None,
+    acl: list[str] | None = None,
+    system_annotations: Mapping[str, Any] | None = None,
+    properties: Mapping[str, Any] | None = None,
+):
```

Comment:
Is it possible to hint a return type for this and PolyNode? I see we are getting the subclass from the label so may not make sense.

Reply:
Doh! That was a miss and forgot docstrings. Added tho!